src/index.html

```diff
@@ -7,6 +7,433 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="moyo" class="header__logo is-active">
+        <img src="images/logo.png" alt="moyo"
+        class="logo-main">
+      </a>
+
+      <nav class="header__navigation">
+        <ul class="header__navigation-menu">
+          <li class="header__navigation-menu-item">
+            <a href="Apple" class="header__navigation-menu-link is-active">Apple</a>
+          </li>
+
+          <li class="header__navigation-menu-item">
+            <a href="Samsung" class="header__navigation-menu-link">Samsung</a>
+          </li>
+
+          <li class="header__navigation-menu-item">
+            <a href="Smartphones" class="header__navigation-menu-link">Smartphones</a>
+          </li>
+
+          <li class="header__navigation-menu-item">
+            <a
+              href="Laptops & Computers"
+              class="header__navigation-menu-link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__navigation-menu-item">
+            <a href="Gadgets" class="header__navigation-menu-link">Gadgets</a>
+          </li>
+
+          <li class="header__navigation-menu-item">
+            <a href="Tablets" class="header__navigation-menu-link">Tablets</a>
+          </li>
+
+          <li class="header__navigation-menu-item">
+            <a href="Photo" class="header__navigation-menu-link">Photo</a>
+          </li>
+
+          <li class="header__navigation-menu-item">
+            <a href="Video" class="header__navigation-menu-link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="card">
+      <div class="card__item"
+      data-qa="card">
+        <img
+        src="./images/imac.jpeg"
+        alt="product-image"
+        class="card__img"
+      >
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__product-code">
+          Product code: 195434
+        </p>
+
+        <div class="card__rang">
+          <div class="card__stars stars-4">
+            <div class="element"></div>
+            <div class="element"></div>
+            <div class="element"></div>
+            <div class="element"></div>
+            <div class="element"></div>
+          </div>
```

Comment:
```suggestion
          <div class="stars card__stars stars--4">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>
```
just reuse the stars block from the previous task
use `card__stars` BEM mix only if you need

`element` is an element of what?